### ISSUE TITLE
ability to define vars without being overridden

### ISSUE THREAD

**Issue by chasebolt** [devspace-cloud/devspace#951]
**Is your feature request related to a problem?**  
I would like to define vars without the user being able to override them via env variables.

**Which solution do you suggest?**  
adding a `source: none` which relies only on `default:` for its value.

**Which alternative solutions exist?**  
This sorta works but it allows the user the ability to override the var with an env variable.

```yaml
vars:
  - name: DEVSPACE_APP_NAME
    default: myapp
    source: env
```

**Additional context**  
Using a variable to pass through to dependencies for dynamic helm values. I cant use profiles as it wouldnt allow the dependency to be as dynamic - would require me to create a profile for every combination.

/kind feature

**Comment by FabianKramm** [comment]
@chasebolt thanks for opening this issue! This sounds useful, we should do that!

**Comment by FabianKramm** [comment]
Implemented with #973
